tests/scripts/thread-cert/node.py

```diff
@@ -1021,7 +1021,7 @@ def remove_route(self, prefix):
         self._expect('Done')
 
     def register_netdata(self):
-        self.send_command('netdataregister')
+        self.send_command('netdata register')
```

Comment:
There should be many other scripts (e.g. OpenThread.py) that uses `netdataregister` and should be updated.